Server/controllers/User/logIn.js

```diff
@@ -0,0 +1,88 @@
+require('dotenv').config();
+const bcrypt = require('bcrypt');
+const { sign } = require('jsonwebtoken');
+const { validateEmail } = require('../../helpers/Validation');
+
+const { SECRET } = process.env;
+const { logIn } = require('../../database/query/User/index');
+const {
+  successMessage,
+  internalErrorMessage,
+  faildLoginMessage,
+  faildMessage,
+} = require('../../helpers/responseMessage');
+
+// To create user token
+const createToken = (email, gid, display_name, secret) =>
+  sign({ email, gid, display_name }, secret);
+
+module.exports = (req, res) => {
+  const { email, password } = req.body;
+  const { error } = validateEmail({ email });
+
+  // If user type his/her email incorrectly
+  if (error !== undefined) {
+    return res
+      .status(403)
+      .json(
+        faildLoginMessage(
+          null,
+          'Make sure that you type your email in a correct way',
+        ),
+      );
+  }
+
+  // If user doesn't enter his/her email or password or both of them
+  if (!email || !password) {
+    return res.status(400).json(faildMessage(null, 'All Fields are Required!'));
+  }
```

Comment:
you can replace all validation  with one joi validation

Reply:
Done